scripts/config/lib/nginx_config_util.rb

```diff
@@ -44,4 +44,16 @@ def self.match_redirects(redirects, uri)
 
     false
   end
+
+  def self.interpolate(string, vars)
+    regex = /\${(\w*?)}/
+
+    string.scan(regex).inject(string) do |acc, capture|
+      var_name = capture.first
+      value = vars[var_name] if vars
+      acc.sub!("${#{var_name}}", value) if value
```

Comment:
> f given as a String, any regular expression metacharacters it contains will be interpreted literally, e.g. '\\d' will match a backlash followed by ‘d’, instead of a digit.

looks like we get `escape` for free. \o/